app/src/main/java/com/duckduckgo/app/browser/WebViewVersionProvider.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser
+
+import android.content.Context
+import androidx.webkit.WebViewCompat
+import com.duckduckgo.app.browser.WebViewVersionProvider.Companion.WEBVIEW_UNKNOWN_VERSION
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import javax.inject.Inject
+
+interface WebViewVersionProvider {
+    companion object {
+        const val WEBVIEW_UNKNOWN_VERSION = "unknown"
+    }
+
+    fun get(): String
+}
+
+interface RawWebViewVersionProvider {
```

Comment:
I don't see why we have 2 interfaces with one implementation each. Should not be better to have 1 interface with 2 implementations? what do you think?

Reply:
we talked about this, and clarified why we had 2 different interfaces. Decided to have an interface(provider) with 2 methods (full webview version and major version), and to create another interface to wrap the access to webviewCompat (datasource)